### PR TITLE
Replace icons with Microsoft Visual Studio Image Library ones

### PR DESCRIPTION
**Pull request by LukaszMendakiewicz**

### CODE REVIEW

File: GoogleTestAdapter/Packaging.GTA/source.extension.vsixmanifest [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <Description xml:space="preserve">Adds support for the C++ unit testing framework Google Tests.</Description>
     <MoreInfo>https://github.com/csoltenborn/GoogleTestAdapter</MoreInfo>
     <License>LICENSE.txt</License>
-    <Icon>Resources\gtest-logo.gif</Icon>
+    <Icon>Resources\Icons\Icon.ico</Icon>
```
Comment (spebl) [inline GoogleTestAdapter/Packaging.GTA/source.extension.vsixmanifest:9]: Should this be "..\Resources\Icons\Icon.ico" like the others?

File: GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <MoreInfo>https://go.microsoft.com/fwlink/?linkid=848768</MoreInfo>
     <License>use.txt</License>
     <ReleaseNotes>ReleaseNotes.txt</ReleaseNotes>
-    <Icon>Resources\gtest-logo.gif</Icon>
+    <Icon>Resources\Icons\Icon.ico</Icon>
```
Comment (spebl) [inline GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest:10]: Same as above.

**Review by spebl — APPROVED** [review]

**Comment by RaulPerez1** [comment]
Icon.ico is a bit too generic, I would call that one GoogleTest.icon

**Review by RaulPerez1 — APPROVED** [review]
Leave a comment

File: GoogleTestAdapter/Packaging.GTA/source.extension.vsixmanifest (hunk 1 above)
Comment (LukaszMendakiewicz) [reply GoogleTestAdapter/Packaging.GTA/source.extension.vsixmanifest:9]: This has to be the path inside VSIX as is generated, and Icon.ico is copied to Resources\Icons\Icon.ico, so this is OK.

File: GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest (hunk 2 above)
Comment (LukaszMendakiewicz) [reply GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest:10]: Ditto.

File: GoogleTestAdapter/Packaging.GTA/source.extension.vsixmanifest (hunk 1 above)
Comment (spebl) [reply GoogleTestAdapter/Packaging.GTA/source.extension.vsixmanifest:9]: Sounds good!

**Pull request merged**